api-reference/openapi.json

```diff
@@ -3058,6 +3182,80 @@
             }
           }
         }
+      },
+      "ProjectCounters": {
```

Comment:
I'd recommend adding an "example" field for schemas (and even for individual properties at some point), so that the Mintlify UI can show a custom sample value in the playground and even auto-fill it for you.

See: https://github.com/edgee-cloud/docs/blob/main/api-reference/data-collection/openapi.json#L166